contrib/benitlux/src/server/benitlux_db.nit

```diff
@@ -173,7 +173,7 @@ DISTINCT day FROM daily WHERE beer=(SELECT ROWID FROM beers WHERE name="{{{beer.
 	fun subscribers: Array[String]
 	do
 		var subs = new Array[String]
-		for row in select("email FROM subscribers") do subs.add row[0].to_s
+		for row in select("email FROM subscribers").as(not null) do subs.add row[0].to_s
```

Comment:
This method should not assert. You are be better off to leave the warning as a reminder so I can fix it the next time I'll open this module.